docs/reference/method/MongoDBGridFSBucket-registerGlobalStreamWrapperAlias.txt

```diff
@@ -57,6 +58,9 @@ revision is read (revision ``-1``).
 In write mode, the stream context can contain the option ``gridfs['chunkSizeBytes']``.
 If omitted, the defaults are inherited from the ``Bucket`` instance option.
 
+The functions `rename` and `unlink` will rename or remove all revisions of a
+filename. If the filename does not exist, these functions throw a ``FileNotFoundException``.
```

Comment:
@jmikola removing or renaming a single revision is not supported. Even if the option `gridfs[revision]` is set.